### DISCUSSION TITLE
Mamba F722 MK3 mit GPS und Kompass

### DISCUSSION THREAD

**Discussion by Mulli311** [iNavFlight/inav#6953] [General]
Hallo habe den MambaF722 MK3 und  BN-880 mit Kompass .Es Funktioniert alles so weit nur Bekomme ich den Kompass nicht auf blau Habe in auf SDL SCD angeschlossen .Kann mir Jemand helfen . PS. Bin Copter Anfänger.!!!!!!!
Kann auch fast kein Englisch .

Mit freundlichem Gruß 
Andreas Müller

**Comment by JulioCesarMatias** [comment]
Inav does not yet support i2c for this FC

**Comment by DzikuVx** [comment]
Inav 3.0 RC1 has mambaf722_i2c target

niedz., 16 maj 2021, 02:40 użytkownik Julio Cesar Matias <
***@***.***> napisał:

**Comment by FHoevi** [comment]
Dear Andreas,
the following CLI commands, by courtesy of Diatone, helped me to get I2C sensors (HMC5883L Mag and MS5611 Baro on a GY-87, my GPS is an additional BN-180) running on my Mamba F722S board (most likely not MK3):

resource SERIAL_TX 3 NONE
resource SERIAL_RX 3 NONE
resource I2C_SCL 2 B10
resource I2C_SDA 2 B11
set dashboard_i2c_bus = 2
set mag_hardware = AUTO
set baro_hardware = AUTO

This means that the usual UART3 pads get transformed into an I2C bus connection (SDA/SCL) which you could use with standard MAMBAF722 target firmware. The GPS sensor of BN-880 still needs to get connected to another free UART port (TX/RX pinout)

Nun auf Deutsch:
Mit den obigen CLI Befehlen, die freundlicherweise von Diatone bereitgestellt wurden, konnte ich meine externen I2C-Sensoren (s.o.) auf meinem Mamba F722 board ans Laufen bekommen. Dadurch wird der vormalige UART3 zu einem I2C Anschluss (SDA/SCL). Ich betreibe mein Board mit der Standard MAMBAF722 target firmware. Der GPS-Sensor des BN-880 muss aber weiterhin über einen weiteren freien UART port (TX/RX) verbunden werden.

**Comment by FHoevi** [comment]
Please note / bitte beachten: #6774
